src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -250,12 +263,16 @@ public void exportFullpages(String query, Integer startYear, Integer endYear, In
         Stream<SolrDocument> docs = streamSolr(finalQuery, max);
         // Create metadata file, that has to be added to output zip
         Map<String, Object> metadataMap = makeMetadataMap(query, startYear, endYear);
+
+        Set<String> fields = new HashSet<>(Arrays.asList("pageUUID", "familyId", "lplace", "fulltext_org"));
```

Comment:
Having a pre-chosen and static setup for the CSV export is fine here. Please extend the description in the `openapi.yaml` to describe the CSV as well as a suggestion for using the `/aviser/export/fields` if more control is needed.